report.py

```diff
@@ -227,7 +227,13 @@ def run_report(config, blockers, server, header, test, save):
 			smtp.ehlo_or_helo_if_needed()
 
 			# send email to all addresses
-			smtp.sendmail(msg['From'], recipients, msg.as_string())
+			response = smtp.sendmail(msg['From'], recipients, msg.as_string())
+
+			# log success if all recipients recieved report, otherwise raise exception
+			if response == {}:
+				print("Report successfully delivered to all recipients")
```

Comment:
This doesn't necessarily mean it's been delivered. 

Better to say that the mail server has accepted the message for delivery.

Reply:
Done, change will be in next patchset